scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/JobData.java

```diff
@@ -84,17 +84,17 @@
                 @NamedQuery(name = "countJobDataFinished", query = "select count (*) from JobData where status = 3"),
                 @NamedQuery(name = "countJobData", query = "select count (*) from JobData"),
                 @NamedQuery(name = "findUsersWithJobs", query = "select owner, count(owner), max(submittedTime) from JobData group by owner"),
-                @NamedQuery(name = "getJobsNumberWithStatus", query = "select count(*) from JobData where status in (:status) and removedTime = -1"),
-                @NamedQuery(name = "getJobsNumberWithStatusUsername", query = "select count(*) from JobData where owner = :username and status in (:status) and removedTime = -1"),
+                @NamedQuery(name = "getJobsNumberWithStatus", query = "select count(*) from JobData where status in (:status)"),
+                @NamedQuery(name = "getJobsNumberWithStatusUsername", query = "select count(*) from JobData where owner = :username and status in (:status)"),
                 @NamedQuery(name = "getJobSubmittedTime", query = "select submittedTime from JobData where id = :id"),
                 @NamedQuery(name = "getMeanJobExecutionTime", query = "select avg(finishedTime - startTime) from JobData where startTime > 0 and finishedTime > 0"),
                 @NamedQuery(name = "getMeanJobPendingTime", query = "select avg(startTime - submittedTime) from JobData where startTime > 0 and submittedTime > 0"),
                 @NamedQuery(name = "getMeanJobSubmittingPeriod", query = "select count(*), min(submittedTime), max(submittedTime) from JobData"),
-                @NamedQuery(name = "getTotalJobsCount", query = "select count(*) from JobData where removedTime = -1"),
+                @NamedQuery(name = "getTotalJobsCount", query = "select count(*) from JobData"),
                 @NamedQuery(name = "loadInternalJobs", query = "from JobData as job where job.id in (:ids)"),
-                @NamedQuery(name = "loadJobs", query = "select id from JobData where status in (:status) and removedTime = -1"),
-                @NamedQuery(name = "loadJobsWithPeriod", query = "select id from JobData where status in (:status) and removedTime = -1 and submittedTime >= :minSubmittedTime"),
-                @NamedQuery(name = "loadJobDataIfNotRemoved", query = "from JobData as job where job.id in (:ids) and job.removedTime = -1"),
+                @NamedQuery(name = "loadJobs", query = "select id from JobData where status in (:status)"),
+                @NamedQuery(name = "loadJobsWithPeriod", query = "select id from JobData where status in (:status) and submittedTime >= :minSubmittedTime"),
+                @NamedQuery(name = "loadJobDataIfNotRemoved", query = "from JobData as job where job.id in (:ids)"),
```

Comment:
@fviale this query in particular requires that the Job must not be removed (removedTime=-1) to fetch its data. (at least what the name says). do you think it is Ok to remove it from here as well ?

Reply:
I should rename the query